public/app/features/plugins/admin/components/Badges/PluginUpdateAvailableBadge.tsx

```diff
@@ -12,13 +12,7 @@ type Props = {
 
 export function PluginUpdateAvailableBadge({ plugin }: Props): React.ReactElement | null {
   const styles = useStyles2(getStyles);
-
-  // Currently renderer plugins are not supported by the catalog due to complications related to installation / update / uninstall.
-  if (plugin.hasUpdate && !plugin.isCore && plugin.type !== PluginType.renderer) {
-    return <p className={styles.hasUpdate}>Update available!</p>;
-  }
-
-  return null;
+  return <p className={styles.hasUpdate}>Update available!</p>;
```

Comment:
Is this change intended for this PR? It doesn't look angular related.

Reply:
This isn't angular related but it caused additional spacing to be present when there was no update. By moving the check to the caller and not rendering anything there, the spacing was correct

Reply:
```
<PluginUpdateAvailableBadge plugin={plugin} />
```

👇 

```
{hasUpdate && <PluginUpdateAvailableBadge plugin={plugin} />}
```

I am referring to this in [PluginListItemBadges.tsx](https://github.com/grafana/grafana/pull/69084/files#diff-2de8bc242744d4ce83055ef3d1105c140cc5d8c735e07dca9b3d80a5953433a8)